org.opent2t.sample.lamp.superpopular/test/lampTests.js

```diff
@@ -139,10 +139,10 @@ function runLampTests(settings) {
     test.serial('SetColourChroma', t => {
         return helpers.runTest(settings, t, () => {
             return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesColourChroma', [deviceId]).then((initialTemperature) => {
-                return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesColourChroma', [deviceId, { 'ct': 30 }]).then(() => {
+                return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesColourChroma', [deviceId, { 'ct': 2500 }]).then(() => {
```

Comment:
revert changes.

---
In reply to: [104008098](https://github.com/openT2T/translators/pull/189#discussion_r104008098) [](ancestors = 104008098)